lucene/CHANGES.txt

```diff
@@ -266,6 +266,9 @@ Bug Fixes
 * LUCENE-9362: Fix equality check in ExpressionValueSource#rewrite. This fixes rewriting of inner value sources.
   (Dmitry Emets)
 
+* LUCENE-9405: IndexWriter incorrectly calls closeMergeReaders twice when the merged segment is 100% deleted.
+  (Simon Willnauer, Mike Mccandless, Mike Sokolov)
```

Comment:
Maybe also add @msfroh (Michael Froh)?  -- I think he fixed this originally in the first PR.

Reply:
yeah, and you can remove me - I am just a conduit!

Reply:
I added him, thanks for pointing this out.

Reply:
I will keep you anyways @msokolov